spec/features/expenses_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Expenses', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Racha Oba', email: 'racha@gmail.com', password: '123456',
+                        password_confirmation: '123456', confirmation_token: nil, confirmed_at: Time.now)
+    @group = Group.create(name: 'Food', icon: 'icons/food.PNG', user_id: @user.id)
```

Comment:
* I get errors when I try to run your tests seems the errors come from the icons in your test as they are different from what you have in your assets folder Kindly put in the correct image title and extension and it should work for example( 'icons/food.jpg') 
![image](https://github.com/rachaouldbabaali/budget-app/assets/52497246/74cba160-a0b6-4d3e-9644-4e9ce33140bb)